ckanext/harvest/templates/source/new_source_form.html

```diff
@@ -37,7 +37,7 @@
         <label class="radio">
           <input type="radio" name="source_type" value="{{ harvester['name'] }}" {{ "checked " if checked }} data-module="harvest-type-change">
           {{ harvester['title'] }}
-          <i class="fa fa-question-circle icon-question-sign muted" title="{{ harvester['description'] }}" data-toggle="tooltip"></i>
+          <i class="fa fa-question-circle icon-question-sign muted" title="{{ harvester['description'] }}" data-bs-toggle="tooltip"></i>
```

Comment:
If the only changes needed to support BS5 are these two small data attributes I think it's overkill to duplicate the whole templates directory. Let's just duplicate the data attributes with the BS3 and BS5 versions and the same template can be used in both versions (I assume each version will ignore the other data attribute):

```suggestion
          <i class="fa fa-question-circle icon-question-sign muted" title="{{ harvester['description'] }}" data-bs-toggle="tooltip" data-toggle="tooltip"></i>
```